### DISCUSSION TITLE
azureAutoComplete vs platformAutomerge

### DISCUSSION THREAD

**Discussion by nhuethmayr** [renovatebot/renovate#30218] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure DevOps

### Please tell us more about your question or problem

Hey, 

I've just recently started using the `azureAutoComplete` flag in my Renovate config as mentioned [here].(https://github.com/renovatebot/renovate/discussions/12925)

Today, I found a Renovate created PR to migrate my config. Crucially, the `azureAutoComplete`  flag is no longer present, the PR itself also is not set to auto-complete🤔

Did `azureAutoComplete` change its name to `platformAutomerge`?

Thanks,
Norbert

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Yes

**Comment by nhuethmayr** [comment]
@rarkins Thanks for your response.

I ran into another little issue, and was wondering if that can be accomplished using Renovate.
Azure, by default copies the description of the PR into the merge message.

![image](https://github.com/user-attachments/assets/14573432-63c8-42a4-8ee2-b5e081bf239f)

Renovate, on the other hand, seems to use the PR title only.
![image](https://github.com/user-attachments/assets/f4b4b4a0-8fff-4ba2-8f82-6fb0ece2477a)

Is there a configuration option to make Renovate stick to Azure DevOps defaults when auto-merging?
